master/static/srv/move_experiment.sql

```diff
@@ -1,3 +1,43 @@
-UPDATE experiments SET project_id = $2
-WHERE id = $1
-RETURNING id;
+WITH update_mn AS (
```

Comment:
I'd rather call `updateProjectHyperparameters` in the same transaction as the old query. this'd be much better for the code readability.

Reply:
This has been fixed